data/api/client-server/report_content.yaml

```diff
@@ -29,11 +29,20 @@ securityDefinitions:
 paths:
   "/rooms/{roomId}/report/{eventId}":
     post:
-      summary: Reports an event as inappropriate.
+      summary: Reports an event as inappropriate. You must be joined to the room in order
+      to report an event.
       description: |-
         Reports an event as inappropriate to the server, which may then notify
         the appropriate people.
+        
+        It may be possible for clients to deduce whether an event exists by
+        timing the response, as only a report for an event that does exist
+        will require the homeserver to check whether a user is joined to
+        the room. To combat this, homeserver implementations may add
+        a random delay when generating a response.
       operationId: reportContent
+      tags:
+        - Reporting content
```

Comment:
out of interest, why did this get moved up? We try to keep it at the bottom for consistency (though this isn't a real requirement)

Reply:
I've moved this back to the bottom of the file.